index.html

```diff
@@ -10,44 +10,35 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!-- <header aria-label="navigation" role="navigation">
-      <div><a href="#" class="welcome">Welcome</a></div>
-      <nav role="navigation">
-        <ul>
-          <li><a class="menu-bar" href="#">Portfolio</a></li>
-          <li><a class="menu-bar" href="#">About</a></li>
-          <li><a class="menu-bar" href="#">Contact</a></li>
-        </ul>
-      </nav>
-      <div class="menu-image"><img class="menu-icon" src="img/Union.png" alt="menu bar" /></div>
-      <div><img class="af image-a" src="img/af.png" alt="background  image design" /></div>
-    </header> -->
-    <nav class="navbar navbar-expand-sm navbar-light  d-flex justify-content-center" aria-label="navigation" role="navigation">
-      <div class="container m-3 m-sm-0">
-        <a href="#" class="navbar d-sm-none navbar-brand welcome"> Welcome</a>
-        <button class="navbar-toggler" 
-        type="button" 
-        data-bs-toggle="collapse"
-        data-bs-target="#nav-menu">
-        <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse justify-content-center" id="nav-menu">
-          <ul class="navbar-nav">
-            <li class="nav-item">
-              <a href="#portfolio" class="nav-link">Portfolio</a>
+
+        <!-- Navigation Section  -->
+
+    <header>
+      <nav class="navbar">
+        <a href="" class="nav-branding welcome">Welcome</a>
```

Comment:
* As shown in the design template kindly hide the welcome text when rendering your drop-down menu 
* ## Given Design 
![image](https://user-images.githubusercontent.com/52497246/208592754-d78bbcbb-e8e1-40eb-841c-045a084f66c4.png)
## Yours 
![image](https://user-images.githubusercontent.com/52497246/208592801-da0068ed-d0ff-4004-8b58-4cade77a236d.png)

Reply:
solved by setting the display to hidden